galaxy/worker/tasks.py

```diff
@@ -317,20 +317,21 @@ def _update_quality_score(import_task):
         content.save()
         repo_points += content.quality_score
 
-        LOG.debug('content - ids, weights, score: {}, {}, {}'.format(
+        # print vs LOG.debug has cleaner celery + recompute_quality log output
+        print('content - ids, weights, score: {}, {}, {}'.format(
```

Comment:
To be discussed.

Reply:
I switched this back to logger and fixed the underlying issue #1288